pkg/monitoring/observability/rules/alerts/cluster_alerts.go

```diff
@@ -0,0 +1,51 @@
+package alerts
+
+import (
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/ptr"
+)
+
+func clusterAlerts() []promv1.Rule {
+	return []promv1.Rule{
+		{
+			Alert: "HighCPUWorkload",
+			Expr:  intstr.FromString("instance:node_cpu_utilisation:rate1m >= 0.9"),
+			For:   ptr.To(promv1.Duration("5m")),
```

Comment:
this should be For: ptr.To[promv1.Duration]("5m") according to https://github.com/kubevirt/hyperconverged-cluster-operator/pull/3482/files#diff-f2a22bd021f6fbfce3c0d37c4b1567df49ae73c88db4a256b5388e5af2ecd785

Reply:
updated